app/controllers/posts_controller.rb

```diff
@@ -174,6 +174,21 @@ def delete_history
   end
 
   def stats
+    post_location = @post.board.name
+    post_location += ' » ' + @post.section.name if @post.section.present?
+    post_location += ' » Stats'
+
+    post_description = generate_short(@post.description)
+    post_description += ' ('
+    post_description += helpers.author_links(@post, linked: false)
+    post_description += ')'
+    post_description.strip!
+
+    @meta_og = {
+      title: @post.subject + ' · ' + post_location,
+      description: post_description,
+      url: @meta_canonical
```

Comment:
I think this variable isn't set to anything in posts#stats, is it?

Reply:
idk, how does that get set?

Reply:
I could only see it declared in `def show_post(cur_page=nil)` on writable_controller – which I think we don't call for the stats page

Reply:
Based on the test at `spec/controllers/post_controller_spec.rb`, line 844 it should be set.

Reply:
That line's not showing anything relevant for me: https://github.com/Marri/glowfic/blob/aa2ceed9e72deca3788f0b6e892a1b9dd7d40de5/spec/controllers/posts_controller_spec.rb#L844 (maybe it was different at a previous commit?)

Reply:
Whoops, misread it as the :show test - updating with a test and setting the url.